guides/m1x/install/installer-privileges_after.html

```diff
@@ -64,7 +64,7 @@ <h2 id="terms">Terminology</h2>
     <h2 id="privs-after">Setting Privileges and Ownership After You Install Magento</h2>
     <p>If you have installed Magento, you can set file system privileges and ownership as follows:</p>
     <ul><li>For a dedicated Magento server, you set ownership of files and directory as the web server user. You set privileges as 500 (directories) and 400 (files).</li>
-        <li>For a hosted Magento server on which the web server runs as the logged-in username, you set privileges as as 500 (directories) and 400 (files).</li></ul>
+        <li>For a hosted Magento server on which the web server runs as the logged-in username, you set privileges as 500 (directories) and 400 (files).</li></ul>
```

Comment:
@tejash-wagento, Hi
These changes are already fixed and accepted. in this PR: https://github.com/magento/magento2/pull/16644